test-appimagetool.sh

```diff
@@ -1,6 +1,8 @@
 #! /bin/bash
 
-set -xe
+set -e
+set -E
+set -o functrace
```

Comment:
What does this do? Eager to learn

Reply:
The `functrace` stuff is needed by the exit hook to be able to display the line number in which it failed. It's not necessarily required.

Reply:
:+1: was just asking to understand it - it should stay, but a comment wouldn't hurt.